test/es6/proxyconstruction.js

```diff
@@ -116,9 +116,9 @@ var tests = [
         }
     },
     {
-        name: "Proxy of Object literal method should be constructable",
+        name: "Proxy of Object literal function should be constructable",
         body() {
-            let testingMethod = {test() {}}
+            let testingMethod = {test:function() {}}
```

Comment:
Could you add another test case here, that is the original code but now says it should fail?

```javascript
        name: "Proxy of Object literal method should not be constructable",
        body() {
            let testingMethod = {test() {}}
            testConstructProxy(testingMethod.test, true);
        }
    },
```